format/bson/testdata/test.fqtest

```diff
@@ -1,79 +1,175 @@
-# https://github.com/dwight/bsontools
-# echo '{array: [1,2,3], object: {key: "value"}, number: 123, string: "abc", true: true, false: false, null: null}' | fromjson test.bson
+# Run the code at https://go.dev/play/p/8YuESN04ACr and copy the base64 output. Then run:
+#
+#   echo "<base64 output>" | base64 -D > test.bson
+#
```

Comment:
I think it makes sense to add the generate program to a file under `format/bson/testdata`. Some reason it has to go thru base64 btw?

Reply:
Also thanks for good tests

Reply:
Aha makes sense